### DISCUSSION TITLE
mDNS

### DISCUSSION THREAD

**Discussion by Sithuk** [zbm-dev/zfsbootmenu#669] [General]
The remote access guide notes that the network enabled image will not advertise itself via dynamic dns or mdns. What is the correct approach to integrate mdns into remote access?

**Comment by ahesford** [comment]
There is no correct approach, because we don't recommend trying to make mDNS work. You'd have to copy whatever mDNS responder (probably Avahi) you're using on the host into the ZFSBootMenu image, possibly with some configuration files to tell it what to announce, make sure you set the hostname as you expect, and launch the daemon at the appropriate time in something like an early-setup hook.

If you want a discoverable boot manager, use real DNS and either a static IP address or some DHCP DDNS magic on the router.
